### ISSUE TITLE
Implement VoiceOver Accessibility Support

### ISSUE THREAD

**Issue by pauljadam** [hakimel/reveal.js#457]
Try the live demo and then run it with VoiceOver. You'll see that VO focus goes through all the hidden elements of the page when it should be constrained to only what is visible in the current slide.
